schedule/_posts/2016-11-30/2016-11-30-11.md

```diff
@@ -23,5 +21,5 @@ GeoSolutions is supporting GFDRR Innovation Lab to meet these needs through:
 * Deployments of GeoNode.
 * Provisioning of GeoNode trainings to developing countries.
 
-In this presentation we will cover the latest developments which we have performed as well as other developments that are planned to gather community feedback and build awareness.
+In this presentation we will cover the latest developments which we have performed, along with technical designs and architectural improvements adopted in GeoNode, as well as related GeoNode Improvement Proposals (GNIPs) to gather community feedback and build awareness.
```

Comment:
Rewrite existing block as:

In this presentation we will cover the GeoNode developments we have performed as well as those which are planned: we will make use of and refer to technical designs and architectural improvements adopted in GeoNode, as well as to the related GeoNode Improvement Proposals (GNIPs) which provide an instrument to gather community feedback and build awareness.

Developments that have been performed and/or  are planned on GeoNode outside the support activities to the GFDRR World Babnk group will also be covered.